$LAAS: patch-ac 2008/09/13 13:33:21 tho $

Fix detection of tcl on x86_64-linux*

--- configure~	2007-08-31 17:09:01.000000000 +0200
+++ configure	2008-09-13 13:32:09.000000000 +0200
@@ -21242,6 +21242,14 @@
 
 
 
+case $host_cpu-$host_os in
+ x86_64-linux*)
+  libsuffix=64
+  ;;
+ *)
+  libsuffix=
+  ;;
+esac
 
 # Check whether --with-tcl or --without-tcl was given.
 if test "${with_tcl+set}" = set; then
@@ -21254,10 +21262,10 @@
          /usr/local/lib/tcl8.3   \
          /usr/local/lib          \
          /usr/pkg/lib            \
-         /usr/lib                \
-         /usr/lib/tcl8.4         \
-         /usr/lib/tcl-8.4         \
-         /usr/lib/tcl8.3         \
+         /usr/lib${libsuffix}    \
+         /usr/lib${libsuffix}/tcl8.4         \
+         /usr/lib${libsuffix}/tcl-8.4         \
+         /usr/lib${libsuffix}/tcl8.3         \
         ; \
        do
          if test -r "$ac_dir/tclConfig.sh"; then
@@ -21328,7 +21336,7 @@
       $TCL_EXEC_PREFIX/lib                    \
       $TCL_PREFIX/lib                         \
       /usr/local/lib                          \
-      /usr/lib                                \
+      /usr/lib${libsuffix}                                \
       /Library/Frameworks/Tcl.framework       \
       $extra_lib                              \
       ; \
